dts/bindings/display/ilitek,ili9xxx-common.yaml

```diff
@@ -35,3 +35,7 @@ properties:
     description:
       Display inversion mode. Every bit is inverted from the frame memory to
       the display.
+
+  mipi-mode:
```

Comment:
You don't need to add this- the property already exists in `mipi-dbi-device.yaml`:

https://github.com/zephyrproject-rtos/zephyr/blob/86af9c4b0766842aa97b5b9bead12c71ca5d69ed/dts/bindings/mipi-dbi/mipi-dbi-device.yaml#L16